cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Hi, @rafaabc,

Why did you add this fixture if it's not being used?

Reply:
@wlsf82 It was created during the setup of Cypress and I'm going to ignore this file in a new commit.

Reply:
Do not put it on `.gitignore` if that's what you meant. I'd simply delete it.

Reply:
It makes sense. I'll delete it as it won't be used

Reply:
Done!